<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/"><channel><title>transaction on Jürgen Gmach</title><link>https://jugmac00.github.io/tags/transaction/</link><description>Recent content in transaction on Jürgen Gmach</description><generator>Hugo -- gohugo.io</generator><language>en-us</language><lastBuildDate>Mon, 15 Feb 2021 13:59:42 +0200</lastBuildDate><atom:link href="https://jugmac00.github.io/tags/transaction/index.xml" rel="self" type="application/rss+xml"/><item><title>What Is the Difference Between Transaction Abort and Transaction Doom</title><link>https://jugmac00.github.io/til/what-is-the-difference-between-transaction-abort-and-transaction-doom/</link><pubDate>Mon, 15 Feb 2021 13:59:42 +0200</pubDate><guid>https://jugmac00.github.io/til/what-is-the-difference-between-transaction-abort-and-transaction-doom/</guid><description>Zope is using the transaction package to manage - you guess - transactions.
More specifically, a transaction starts when Zope receives a request, and the transaction succeeds when the action triggered by the request works out.
When the action causes an exception, the transaction will be rolled back.
This means, usually you very rarely have to interfere with the transaction management manually.
Last week I was implementing a new XML-RPC API, which basically looks like the following code:</description></item></channel></rss>